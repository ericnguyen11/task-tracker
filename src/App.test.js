import { shallow } from "enzyme";

import App from "./App.js";

it("passes by default", () => {
  expect(true).toBe(true);
});

it("renders without crashing", () => {
  shallow(<App />);
});

it("matches snapshot", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
