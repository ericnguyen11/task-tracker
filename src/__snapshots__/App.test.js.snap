// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
>
  <Header
    onAdd={[Function]}
    showAdd={false}
    title="Task Tracker"
  />
  <Routes>
    <Route
      element={<About />}
      path="/about"
    />
    <Route
      element={
        <React.Fragment>
          No tasks
        </React.Fragment>
      }
      path="/"
    />
  </Routes>
  <Footer />
</div>
`;
